superflore/generators/ebuild/ebuild.py

```diff
@@ -250,9 +244,10 @@ def get_ebuild_text(self, distributor, license_text, die_msg=None):
         ret += "\"\n\n"
 
         # SLOT
-        ret += "SLOT=\"{}\"\n".format(self.distro)
+        ret += "SLOT=\"0\"\n"
         # CMAKE_BUILD_TYPE
-        ret += "CMAKE_BUILD_TYPE=RelWithDebInfo\n"
+        if self.name == "catkin":
+            ret += "BUILD_BINARY=\"0\"\n"
```

Comment:
Is this equivalent to https://github.com/allenh1/superflore/pull/28/files#diff-5c266db61ea0f71dcf9a23aed337bb31L277